### DISCUSSION TITLE
v1.4.5

### DISCUSSION THREAD

**Discussion by fl0ppy-d1sk** [bunkerity/bunkerweb#372] [Help]
Documentation : https://docs.bunkerweb.io/1.4/

Docker tags : 
- `bunkerity/bunkerweb:1.4.5`
- `bunkerity/bunkerweb-autoconf:1.4.5`
- `bunkerity/bunkerweb-ui:1.4.5`

Linux packages : https://packagecloud.io/app/bunkerity/bunkerweb/search?q=1.4.5&filter=all&dist=

Changelog : 
- Fix bwcli syntax error
- Fix UI not working using Linux integration
- Fix missing openssl dep in autoconf
- Fix typo in selfsigned job

<hr /><em>This discussion was created from the release <a href='https://github.com/bunkerity/bunkerweb/releases/tag/v1.4.5'>v1.4.5</a>.</em>

**Comment by alexanderadam** [comment]
@fl0ppy-d1sk probably a stupid question but is there anything to mitigate [the memory leaks of ModSecurity?](https://github.com/SpiderLabs/ModSecurity/issues/2848).

One of the maintainers suggest

> to at least periodically do a true restart

although I find that really awkward.
But the issue is known since roughly two years and it doesn't look like there's a fix soonish.

**PS:** [NGINX moves from ModSecurity WAF to Coraza WAF at one point](https://www.nginx.com/blog/f5-nginx-modsecurity-waf-transitioning-to-eol/). But [Coraza WAF](https://coraza.io/) still seems to be in development and [having ongoing changes](https://github.com/corazawaf/coraza/issues/336).
